### ISSUE TITLE
[Bug]:  ControlNet inpaint "only masked" incorrect cropping

### ISSUE THREAD

**Issue by TimothyKrell** [Mikubill/sd-webui-controlnet#2626]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

When inpainting with controlnet, the sample passed to controlnet does not match the cropped portion of the image. You can see the entire image is being overlayed into the inpaint region. This is on a fresh install of sdnext.

### Steps to reproduce the problem

1. Go to inpaint
2. Mask inpaint region
3. turn on controlnet tile
4. Generate

![302653851-5a0eae65-800f-44df-bcc0-1f61eba53af5](https://github.com/Mikubill/sd-webui-controlnet/assets/1585669/96a2b68c-1baf-47dc-8cf1-6a5723b809a0)
![302653919-ba14f649-5e52-48b2-8dec-b1df46812d51](https://github.com/Mikubill/sd-webui-controlnet/assets/1585669/cc2fe9c5-3d5e-4b36-a374-39a9b6817583)
![302654386-e4beb21e-827a-423c-b79d-17f3da5a3f6a](https://github.com/Mikubill/sd-webui-controlnet/assets/1585669/8a87b1a2-0173-4428-bdf5-a5d1a1f99211)
![302654601-aa44ce05-e7dc-4435-a351-f49afb9f04bb](https://github.com/Mikubill/sd-webui-controlnet/assets/1585669/129a8269-ba29-4d6c-89cb-e796e1035994)

### What should have happened?

Controlnet should receive the correctly cropped region passed to its control image.

### Commit where the problem happens

webui: SDNext (latest)
controlnet: 416c3450

### Command Line Arguments

```Shell
.\webui.bat --models-dir C:\Users\timot\Code\AI\models\
```

### List of enabled extensions

'Lora', 'sd-extension-chainner', 'sd-extension-system-info', 'sd-webui-agent-scheduler', 'sd-webui-controlnet',
                         'stable-diffusion-webui-images-browser', 'stable-diffusion-webui-rembg'

### Console logs

```Shell
https://pastebin.com/gwAJvxHN
```

**Comment by huchenlei** [comment]
So SDNext does not seem to properly display the `Crop input image with A1111 mask` checkbox. In theory that checkbox should be visible when you select inpaint only masked.

**Comment by 2blackbar** [comment]
yup works in auto11, sometimes you have to remove json file with all the config saved cause sometimes it just wont add new lines to config when extension already exists, it bugged me when i tried to add custom boxes to extensions, had to remove config ui json file over and over with each change to extension
